integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -364,18 +364,18 @@ public int open(String path, FuseFileInfo fi) {
   private int openInternal(String path, FuseFileInfo fi, int flags, OpenAction openAction) {
     final AlluxioURI uri = mPathResolverCache.getUnchecked(path);
     if (openAction == OpenAction.NOT_SUPPORTED) {
-      LOG.error(String.format("Failed to open %s: Not supported open flag 0x%x. "
+      LOG.error("Failed to open {}: Not supported open flag {}. "
```

Comment:
These are okay since error level logging is almost always enabled, so the strings need to be created anyway.

Reply:
Got it. Error level logs are left untouched now.